projects/python/simulation/agricultural-simulation-dataset-generator/agricultural-environment-robotti-mavicpro/controllers/generate_robotti_obstacles_datasets/generate_robotti_obstacles_datasets.py

```diff
@@ -0,0 +1,472 @@
+# Copyright 2020-2023 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from controller import Supervisor, Camera, Lidar, GPS
+
+useRobotti = True
+
+if (useRobotti):
```

Comment:
Is there any other case?

Reply:
Basically, the idea was to have the possibility of using both, or either one of them UGV or UAV.